llvm/lib/Target/PowerPC/AsmParser/PPCAsmParser.cpp

```diff
@@ -130,6 +131,8 @@ class PPCAsmParser : public MCTargetAsmParser {
 
   void processInstruction(MCInst &Inst, const OperandVector &Ops);
 
+  bool hasMemOp(const OperandVector &Ops);
```

Comment:
It looks to me that this could be a static function at namespace scope in this `.cpp` file.